code/src/index.css

```diff
@@ -1,13 +1,362 @@
+* {
+	box-sizing: border-box;
+}
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	margin: 0;
+	font-family: "Grenze Gotisch", serif, -apple-system, BlinkMacSystemFont,
+		"Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans",
+		"Droid Sans", "Helvetica Neue", sans-serif;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+	background-color: #010507;
+	color: #fff;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+	font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+		monospace;
+}
+
+/*START BOARD*/
+.start-board {
+	display: flex;
+	flex-direction: column;
+	min-height: 100vh;
+	justify-content: center;
+	background-image: url("./assets/startpage.jpg");
+	background-size: cover;
+	background-position: center;
+	width: 100%;
+}
+
+form {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+.start-board-user-input {
+	width: 100%;
+	height: 45px;
+	background: transparent;
+	color: #f3f3f3;
+	border: 1px solid #1c2023;
+	font-family: "Grenze Gotisch", serif;
+	font-size: 21px;
+	padding: 5px;
+	text-align: center;
+	max-width: 345px;
+}
+
+input::placeholder {
+	color: #d6d4d4;
+}
+
+.start-board-game-title {
+	font-size: 32px;
+	margin-bottom: 20px;
+	font-family: "Grenze Gotisch", serif;
+	text-align: center;
+	color: #d37b04;
+}
+
+/*GAMEBOARD*/
+
+.gameboard-current-details {
+	display: flex;
+	flex-direction: column;
+}
+
+.gameboard-player-name {
+	align-self: flex-end;
+	margin-bottom: 0;
+	margin-right: 10px;
+}
+
+.board-wrapper {
+	display: flex;
+	justify-content: center;
+}
+
+.gameboard-main {
+	background-size: cover;
+	background-position: center;
+}
+
+/*Dark cave*/
+.bg-0-0 {
+	background-image: url("./assets/0-0.jpg");
+}
+/*Cave sunlight*/
+.bg-1-0 {
+	background-image: url("./assets/1-0.jpg");
+}
+/*Colorful cave*/
+.bg-1-1 {
+	background-image: url("./assets/1-1.jpg");
+}
+/*room with mechalical gagdets*/
+.bg-0-1 {
+	background-image: url("./assets/0-1.jpg");
+}
+
+.bg-0-2 {
+	background-image: url("./assets/0-2.jpg");
+}
+
+.bg-0-3 {
+	background-image: url("./assets/0-3.jpg");
+}
+
+.bg-1-3 {
+	background-image: url("./assets/1-3.jpg");
+}
+
+.game-board-wrapper {
+	width: 100%;
+}
+.gameboard {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	min-height: 97vh;
+	background-color: rgb(0 0 0 / 76%);
```

Comment:
Here we learned something new! We didn't know that one could use only rgb and use % to specify opacity!